client/my-sites/ads/controller.js

```diff
@@ -27,7 +29,9 @@ function _recordPageView( context, analyticsPageTitle ) {
 }
 
 function _getLayoutTitle( context ) {
-	var title = sites.getSelectedSite().jetpack ? 'Ads' : 'WordAds';
+	const state = context.store.getState();
+	const siteId = getSelectedSiteId( state );
+	const title = isJetpackSite( state, siteId ) ? 'Ads' : 'WordAds';
```

Comment:
Tangent to this PR, but this is broken i18n 😞, as:
- It doesn't allow translating either of the terms _Ads_ or _WordAds_ (which may be intentional but may be very bad for non-Latin-scripted langs or in langs that attach case markers to foreign nouns).
- It doesn't allow the use of a different sentence depending on the term (_Ads_ or _WordAds_), which may be relevant.
- Provides little context for translators in the `translate` calls.

cc @dbspringer